### ISSUE TITLE
[Animal] Support Multiple Breed

### ISSUE THREAD

**Issue by allisterakun** [RuminantFarmSystems/RuFaS#1681]
1. Make the `Animal.breed` attribute an enum with the following possible values:
    1. [Already-In-Place] HO: Holstein
    2. JE: Jersey
    3. [Nice-To-Have] XX: Cross-breed
    4. [Nice-To-Have] BS: Brown Swiss
    5. [Nice-To-Have] GU: Guernsey
2. Update the following submodule to accommodate different breeds:
    1. `AnimalGrowth`
    2. `MilkProduction`
    3. `AnimalGenetic` (after implementation)
3. When changing the breed, make sure to change the animal input variables for the above submodules for the corresponding breed.
4. [Nice-To-Have] Multiple breeds in a single simulation
    1. Create an extra input for herd composition.
    2. Restructure the input structure for AnimalGrowth-related and MilkProduction-related variables as a list of objects, each representing a breed. All objects in the same list should have the same structure.

**Target Deadline**
By when should this issue be done.

**Comment by allisterakun** [comment]
Blocked by #1680

**Comment by matthew7838** [comment]
Grooming session: @allisterakun Will discuss with Julie and Joe about this issue.

**Comment by ew3361zh** [comment]
Grooming session:
- @allisterakun to follow up with SMEs and will leave note on status after following up.

**Comment by allisterakun** [comment]
Grooming session: Waiting for more SME input on this

**Comment by tomhuhh** [comment]
SME grooming: for now, HO vs JE milk production can use the same equations. We don’t have breed‑specific lactation curve (Wood) parameters. The lactation‑curve parameter refinement work (@YijingGong + me) is likely the current path to generate breed/parity‑specific wood_l/m/n; thus, I’d keep MilkProduction breed‑agnostic.

For “multiple breeds in one simulation”, a minimal, backwards‑compatible approach is to add a herd composition input (e.g., [{breed: HO, fraction: 1.0}] by default) and assign breed during herd initialization; keep default inputs 100% Holstein.

Also, AnimalGenetics appears Holstein‑only, so multi‑breed genetics may need to be explicitly handled in a follow‑up issue. 

Overall this feels lower priority, but happy to hear others’ views.
